CALEXT2_WeekSlot.js

```diff
@@ -136,19 +136,19 @@ class WeekSlot extends Slot {
             continue;
           } else {
             const tlDoms = timelineDom.querySelectorAll(".timelineSleeve");
-            var tlDom = tlDoms[k];
+            const tlDomA = tlDoms[k];
```

Comment:
> reverting https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/commit/524a78687ef19b727da21bae637a31472c13f025 is the solution

Thanks for working it out! :smiley: I have undone that. But is there anything else we can do here besides deactivating the linter rule? :slightly_smiling_face: 

> wouldn't switching from var to const be enough ... ?

Unfortunately not, as they are actually all in the same scope, because if statements do not have their own scope. I tried a different approach and deactivated the variable earlier: https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/pull/158/commits/e7de0f94b12ff8edc4615f081ed1ad85cd98f275 Is that better?

Reply:
> e7de0f9 Is that better?

Yeah, I think so!